Flex Gateway Get Well: add "Configuration" pages (#209)
mulesoft/docs-gateway: gateway/1.4/modules/ROOT/pages/_partials/task-rep-run-flex-gateway.adoc

//tag::add-flex-intro[]
== Flex Replicas

> A Flex Replica is an instance of Flex Gateway. Setting up your gateway layer with multiple Flex Gateway replicas will enable the APIs deployed on Flex Gateway to serve more traffic.

Reviewer: ```suggestion
A Flex Replica is an instance of Flex Gateway. Setting up your gateway layer with multiple Flex Gateway replicas enables the APIs deployed on Flex Gateway to serve more traffic.
```

Reviewer: (present tense)

> When using Flex Gateway to manage APIs in production, you should at least have more than one Flex Replica to avoid a single point of failure. Running multiple Flex Replicas also increases performance if your API service receives a significant amount of traffic.

Reviewer: ```suggestion
When using Flex Gateway to manage APIs in production, configure more than one Flex Replica to avoid a single point of failure. Running multiple Flex Replicas also increases performance if your API service receives a significant amount of traffic.
```
